io.github.philippkosarev.bmi.metainfo.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.philippkosarev.bmi</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <name>BMI</name>
+  <summary>A body mass index calculator</summary>
+  <description>
+    <p>BMI is an open-source, body mass index calculator built with GTK4 and LibAdwaita. </p>
+  </description>
+  <developer id="tld.vendor">
+    <name>Philipp Kosarev</name>
+  </developer>
+  <!-- Requered: Should be a link to the upstream homepage for the component -->
+  <url type="homepage">https://github.com/philippkosarev/bmi</url>
+  <!-- Recommended: It is highly recommended for open-source projects to display the source code repository -->
+  <url type="vcs-browser">https://github.com/philippkosarev/bmi</url>
+  <!-- Should point to the software's bug tracking system, for users to report new bugs -->
+  <url type="bugtracker">https://github.com/PhilippKosarev/bmi/issues</url>
+  <!-- Should link a FAQ page for this software, to answer some of the most-asked questions in detail -->
+  <!-- URLs of this type should point to a webpage where users can submit or modify translations of the upstream project -->
+  <!-- <url type="translate">https://example.org/translate</url> -->
+  <!-- <url type="faq">https://example.org/faq</url> -->
+  <!--Sould provide a web link to an online user's reference, a software manual or help page -->
+  <!-- <url type="help">https://example.org/help</url> -->
+  <!-- URLs of this type should point to a webpage showing information on how to donate to the described software project -->
+  <!-- <url type="donation">https://example.org/donate</url> -->
+  <!--This could for example be an HTTPS URL to an online form or a page describing how to contact the developer -->
+  <url type="contact">https://github.com/PhilippKosarev</url>
+  <!--URLs of this type should point to a webpage showing information on how to contribute to the described software project -->
+  <url type="contribute">https://github.com/PhilippKosarev/bmi</url>
+  <translation type="gettext">bmi</translation>
+  <!-- All graphical applications having a desktop file must have this tag in the MetaInfo.
+     If this is present, appstreamcli compose will pull icons, keywords and categories from the desktop file. -->
+  <launchable type="desktop-id">io.github.philippkosarev.bmi.desktop</launchable>
+  <!-- Use the OARS website (https://hughsie.github.io/oars/generate.html) to generate these and make sure to use oars-1.1 -->
+  <content_rating type="oars-1.1"/>
+  <!-- Applications should set a brand color in both light and dark variants like so -->
+  <branding>
+    <color type="primary" scheme_preference="light">#f6f5f4</color>
+    <color type="primary" scheme_preference="dark">#3a3543</color>
+  </branding>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/PhilippKosarev/bmi/blob/main/screenshot.png</image>
```

Comment:
these need to be direct links https://raw.githubusercontent.com/PhilippKosarev/bmi/0a37115b4cff898ec0a6c54ae5f50ae544816942/screenshot.png